### PR TITLE
DROOLS-5131: Extend coverage of boxed list expression

### PR DESCRIPTION
**Pull request by jomarko**
This PR extends coverage of appending list expression entry
    
    During this coverage extension a new issue was found https://issues.redhat.com/browse/KOGITO-1581
    This PR serves also as reproducer for this new issue.

### CODE REVIEW

**Comment by jomarko** [comment]
Dependent on #3220

File: kie-wb-common-dmn/kie-wb-common-dmn-webapp-kogito-runtime/src/test/java/org/kie/workbench/common/dmn/showcase/client/model/ListSeleniumModel.java [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+
+    private String name;
+
+    private List<String> items;
+
+    public String getName() {
+        return name;
+    }
+
+    public void setName(String name) {
```
Comment (manstis) [inline kie-wb-common-dmn/kie-wb-common-dmn-webapp-kogito-runtime/src/test/java/org/kie/workbench/common/dmn/showcase/client/model/ListSeleniumModel.java]: You could be consistent with your use of `final` in this class.

File: kie-wb-common-dmn/kie-wb-common-dmn-webapp-kogito-runtime/src/test/java/org/kie/workbench/common/dmn/showcase/client/selenium/DMNDesignerKogitoSeleniumIT.java [hunk 2]
```diff
@@ -2160,6 +2179,39 @@ private void setDecisionTableDefaultOutput(final DecisionTableSeleniumModel deci
...
     }
 
+    /**
+     * This method extends coverage of DROOLS-5131
+     */
+    private void appendBoxedListExpressionItem(final ListSeleniumModel list, final String item) {
+        expandDecisionNavigatorDock();
+        final WebElement node = waitOperation().until(element(LIST, list.getName()));
+        assertThat(node)
+                .as("Decision table of '" + list.getName() + "'was not present in the list of nodes")
```
Comment (manstis) [inline kie-wb-common-dmn/kie-wb-common-dmn-webapp-kogito-runtime/src/test/java/org/kie/workbench/common/dmn/showcase/client/selenium/DMNDesignerKogitoSeleniumIT.java]: It's not a "Decision table" in this test.

File: kie-wb-common-dmn/kie-wb-common-dmn-webapp-kogito-runtime/src/test/java/org/kie/workbench/common/dmn/showcase/client/model/ListSeleniumModel.java (hunk 1 above)
Comment (jomarko) [reply kie-wb-common-dmn/kie-wb-common-dmn-webapp-kogito-runtime/src/test/java/org/kie/workbench/common/dmn/showcase/client/model/ListSeleniumModel.java]: Done, thank you

File: kie-wb-common-dmn/kie-wb-common-dmn-webapp-kogito-runtime/src/test/java/org/kie/workbench/common/dmn/showcase/client/selenium/DMNDesignerKogitoSeleniumIT.java (hunk 2 above)
Comment (jomarko) [reply kie-wb-common-dmn/kie-wb-common-dmn-webapp-kogito-runtime/src/test/java/org/kie/workbench/common/dmn/showcase/client/selenium/DMNDesignerKogitoSeleniumIT.java]: Done, thank you

**Comment by jomarko** [comment]
jenkins please retest this

**Comment by jomarko** [comment]
Failed test 
```
org.guvnor.ala.services.rest.tests.RestRuntimeProvisioningImplTest.testAPI
```

**Comment by jomarko** [comment]
Can not reproduce failure locally.

**Comment by jomarko** [comment]
jenkins please retest this

**Comment by jomarko** [comment]
@kiegroup/gatekeepers please merge

**Pull request merged**